src/Components/ElementsPanel/Internal/ElementsList.tsx

```diff
@@ -72,155 +145,131 @@ const ElementsList: React.FC<IViewerElementsPanelListProps> = ({
 
 function getListItems(
     panelItems: Array<IViewerElementsPanelItem>,
+    isModal: boolean,
+    openCalloutRowId: string,
     onItemClick: ElementsPanelCallback,
-    onItemHover?: ElementsPanelCallback,
-    onItemBlur?: ElementsPanelCallback
+    onItemHover: ElementsPanelCallback,
+    onItemBlur: ElementsPanelCallback,
+    onItemHoverExtended: ViewerElementsPanelCallback,
```

Comment:
do we have to pass separate callbacks? Can't we just always pass the extended version and just handle the cases internally to those functions where there isn't a callout?

Reply:
I need to send in both callbacks in because badge rows don't open the tooltip. Adding an argument to open or not the tooltip feels worse than two functions.